.drone.yml

```diff
@@ -73,7 +73,7 @@ steps:
   - name: build_image
     image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/dind
     commands:
-      - n=0; while [ "$n" -lt 60 ] && [ ! docker stats --no-stream ]; do n=$(( n + 1 )); sleep 1; done
+      - n=0; while [ "$n" -lt 60 ] && ! docker stats --no-stream >/dev/null 2>&1; do n=$(( n + 1 )); sleep 1; done
```

Comment:
just curious can you explain this to me?

Reply:
This was ACP's fix for an issue with the dind (Docker in Docker) image for Drone builds. The issue was causing the build_image step to fail in Drone.